test/resources/openmpi/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+FROM mvsusp/openmpi
```

Comment:
Is this the Docker image built from Dockerfile.base? If so, could you document this?

Also: seems odd to have it under a personal namespace -- I think this should either be put in a team-owned repository or assumed that the base is built locally with a certain name.